zilliqa/src/consensus.rs

```diff
@@ -610,6 +615,16 @@ impl Consensus {
             .block_time
             .saturating_sub(milliseconds_since_last_view_change);
 
+        // Override milliseconds_remaining_of_block_time in forced set view scenario
+        match self.force_view {
+            Some((forced_view, timeout_instant)) if view == forced_view => {
+                milliseconds_remaining_of_block_time = SystemTime(timeout_instant.to_system_time())
```

Comment:
Can we add and use `impl From<DateTime> for zilliqa::time::SystemTime` in `time.rs` instead? I'm pretty sure this line of code won't compile when the `fake_time` is not enabled.